test/stringmethods.test.js

```diff
@@ -16,5 +16,105 @@ describe('hasVowel Method Test suite', function () {
         expect('@$%^_.!`+-}{<>'.hasVowels()).toBeFalsy();
     });
 
-}); 
+});
 
+// convert string to upper case test suite
+describe('toUpper Test Suite', function () {
+    test('Returns the String in question but with all characters in upper cases as applicable', function () {
+        expect('Hello'.toUpper()).toBe('HELLO');
```

Comment:
for example, these should be
test('if the toUpper function returns a mixed cased word to upper case')